test/Zapper.ts

```diff
@@ -52,15 +54,19 @@ function numFromBn (num: BigNumber, decimals = 18): number {
     return Number(ethers.utils.formatUnits(num, decimals));
 }
 
+function formatBn (num: BigNumber, decimals = 18): BigNumber {
+    return ethers.utils.parseUnits(num.toString(), 18);
+}
+
 async function getUserSomeWETH (r: ContractTestContext) {
     externalWETH = new ethers.Contract(addressWETH9, abiWETH9Token, owner);
-    await ethers.provider.send("evm_mine");
+    await ethers.provider.send("evm_mine", []);
```

Comment:
Why adding []?

Reply:
I'm getting redlines when there's only 1 argument, I'm assuming my version of ethers requires 2 arguments